### DISCUSSION TITLE
How to connect to Synapse Analytics Workspace

### DISCUSSION THREAD

**Discussion by v-gkhattar** [microsoft/SynapseVSCode#19] [Q&A]
I can only see Fabric workspaces in the list, how can I connect to Synapse Analytics Workspace?

**Comment by rabbyn** [comment]
Hello @v-gkhattar , I'm not from the developer team of this extension but I can easily say that this extension is something only applicable for Fabric Synapse (and not Azure Synapse) ;  see presentation video @ https://www.youtube.com/watch?v=A9SjAyZ_JSc

**Comment by jtdossett** [comment]
Seems like the extension is named poorly.

**Comment by kubrakutlu** [comment]
Will Synapse Analytics support come in future?
